service/core/action/format/delete.go

```diff
@@ -52,6 +52,17 @@ func delete(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	var cnt int
+	config.DB.Model(&model.Post{}).Where(&model.Post{
+		SpaceID:  uint(sID),
+		FormatID: uint(id),
+	}).Count(&cnt)
```

Comment:
@monarkatfactly  I don't think  space_id is required in where query.

Reply:
@shreeharsha-factly I think keeping space ID in query is kind off double security

Reply:
Yes, it narrows down our result to a particular space.

Reply:
@hetjagani check without spaceID if query time decrease then remove it